WordPress/src/main/java/org/wordpress/android/util/KeyboardResizeViewUtil.java

```diff
@@ -0,0 +1,64 @@
+package org.wordpress.android.util;
+
+import android.app.Activity;
+import android.graphics.Rect;
+import android.view.View;
+import android.view.ViewTreeObserver;
+
+/**
```

Comment:
@mzorz so I got this class from Stack Overflow so I left this comment as is to note its source? That's the appropriate way to credit the author right?

Reply:
Sounds good